qacode/core/utils/Utils.py

```diff
@@ -8,6 +8,7 @@
 class Utils(object):
     """All methods what i don't know where to include"""
 
+    @staticmethod
     def get_file(version_py=version_info.major,file_path=None, file_name=None, encoding=None, json=False):
```

Comment:
Lots of things to comment here:
* The method name `get_file` doesn't looks very descriptive, what does it "get"? The file name? The file path? The file contents? A file-like object? `read_file` sounds much less ambiguous.
* Don't include `version_py` as a parameter unless you know a use-case where you want to use Python 2 read-like code in Python 3, or viceversa; just use `sys.version_info.major` instead.
* `file_path` and `file_name` can be joined as a single argument, `file_path`, which would contains the full path to the file; that would also allow you to remove the joining of the file path and name, let the user (e.g. other programmer) join them.
* Instead of reading the file without encoding when `encoding=None`, sounds better to just set `encoding = 'utf8'` (a good, sane default) when it's `None`, then you can delete the code path (`if`) for `encoding=None`.
* The `json` parameter doesn't make much sense, if the user (the programmer who use this method) wants to parse the content as JSON, let him just do it with `json.loads(get_file(...))`, don't pollute that in a method that must just read a file.

With these tips you will have a method like this:

    def read_file(file_path):
        ....

Which is much simpler to use *and* to maintain.

Most of the time, [less is more](https://gettingreal.37signals.com/ch10_Less_Software.php) 😉

Reply:
🥇